win32/devdoc/callonce_requirements.md

```diff
@@ -0,0 +1,87 @@
+# call_once requirements

```

Comment:
>call_once [](start = 2, length = 9)

Can we put all of this (lazy_init too) in common since it's not really win32 specific? One less thing to fix for cross-plat

Reply:
Yout got it at your next comment, interlocked_hl is not there for me, as I need WakeAll.

---
In reply to: [466041572](https://github.com/Azure/azure-c-pal/pull/28#discussion_r466041572) [](ancestors = 466041572)